src/tests/multihost/alltests/test_misc.py

```diff
@@ -480,3 +479,57 @@ def test_0008_1636002(multihost, backupsssdconf):
         assert "Failed to connect to '/var/lib/sss/db/config.ldb'" \
             not in log_str
         assert "The confdb initialization failed" not in log_str
+
+    @staticmethod
+    @pytest.mark.tier1
+    def test_0009_dbus_method_find_usrby_attr(multihost, backupsssdconf, ldap_posix_usergroup):
+        """
+        :title: D-Bus method to find user by attributes
+        :id: ee3437ff-572e-472e-8f55-0d7d8134266c
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2128840
+        :customerscenario: true
+        :setup:
+         1. In sssd.conf
+            set ldap_user_extra_attrs = sn:sn in domain section of sssd.conf
+            set user_attributes = +sn in [ifp] section
+         2. Create 10 users with 'sn' attribute containing pattern foo
+         3. Create a user who does not have sn=foo pattern in it's sn attribute
+        :steps:
+            1. Restart sssd with clean cache
```

Comment:
Please indent setup,  steps, results the same.

Reply:
Added indentation accordingly